packages/SwingSet/src/controller/controller.js

```diff
@@ -466,10 +466,14 @@ export async function makeSwingsetController(
       resultPolicy = 'ignore',
       slots = [],
     ) {
+      // TODO: Use a proper makeMarshal serialize.
```

Comment:
@FUDCo , does your smallcaps PR do this todo? 

@gibson042 @FUDCo is it clear what the merge of this change and @FUDCo 's upcoming smallcaps change should be? Could that be adopted here by itself even if this PR gets merged before the smallcaps PR does?